relational/test_table.py

```diff
@@ -133,6 +134,26 @@ def test_table_project_facts_units(basic_graph):
     # Gets from PTS because permit_type 3 is also valid
     assert _get_value_for_row(table, fields, 'net_num_units') == '7'
 
+    entries7 = [
```

Comment:
If we do end up adding the filtering on status, probably want a test for that? Also a test for de-duping permit numbers?

Reply:
the de-duping test exists, and is for entries7. i might reformat this test a bit because it's getting a bit large to follow